ekncontent/ekncontent/eknc-database-manager.c

```diff
@@ -314,6 +316,8 @@ create_database_from_manifest (const char  *manifest_path,
 {
   GError *error = NULL;
 
+  g_autoptr(EosProfileProbe) probe = EOS_PROFILE_PROBE ("/ekncontent/database/create");
+
```

Comment:
Like we mentioned on slack, this would only happens once in a regular apps so we a) allow profile probe to display the one sample time or b) removed it (since it doesn't provide any info).

Reply:
eos-sdk#4177 should fix this.